hack/golangci-lint.sh

```diff
@@ -8,23 +8,29 @@ set -e
 # a very old version, where modern features (like `declare -A`) are
 # absent.
 
+declare -a EXTRA_TAGS
+
 echo "Linting for GOOS=$GOOS"
-# Special case: for Darwin and Windows only "remote" linting is possible and required.
-if [[ "$GOOS" == "windows" || "$GOOS" == "darwin" ]]; then
-  (
-    set -x
-    ./bin/golangci-lint run --build-tags="remote,containers_image_openpgp" "$@"
-  )
-  exit 0
-fi
+case "$GOOS" in
+  windows|darwin)
+    # For Darwin and Windows, only "remote" linting is possible and required.
+    TAGS="remote,containers_image_openpgp"
+    ;;
+  freebsd)
+    TAGS="containers_image_openpgp"
+    EXTRA_TAGS=(",exclude_graphdriver_btrfs,cni" ",remote")
+    ;;
+  *)
+    # Assume Linux: run linter for various sets of build tags.
+    TAGS="apparmor,seccomp,selinux"
+    EXTRA_TAGS=("" ",systemd" ",remote")
+esac
 
-# Normal case (Linux): run linter for various sets of build tags.
-TAGS="apparmor,seccomp,selinux"
-for EXTRA_TAGS in "" ",systemd" ",remote"; do
+for EXTRA in "${EXTRA_TAGS[@]}"; do
```

Comment:
hold on, this means we are not linting anything one windows  and macos now?!
EXTRA_TAGS is empty and thus doesn't loop at all there unless I am missing something.

Reply:
Yep, I screwed up, thanks for catching!

This is the fix:
```diff
diff --git a/hack/golangci-lint.sh b/hack/golangci-lint.sh
index 39106f96e3..d0c5c58eb6 100755
--- a/hack/golangci-lint.sh
+++ b/hack/golangci-lint.sh
@@ -23,10 +23,10 @@ case "$GOOS" in
   *)
     # Assume Linux: run linter for various sets of build tags.
     TAGS="apparmor,seccomp,selinux"
-    EXTRA_TAGS=("" ",systemd" ",remote")
+    EXTRA_TAGS=(",systemd" ",remote")
 esac
 
-for EXTRA in "${EXTRA_TAGS[@]}"; do
+for EXTRA in "" "${EXTRA_TAGS[@]}"; do
   # Use set -x in a subshell to make it easy for a developer to copy-paste
   # the command-line to focus or debug a single, specific linting category.
   (
```

(pushed)